### ISSUE TITLE
Editing a Fill-Up does not recalculate Cost/l

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#27]
```
What steps will reproduce the problem?
1. Add fillup £70 - 50l = £1.40
2. Correct £70 to £65 and cost still says £1.40

Expected behavior: Cost should be £1.30 and not £1.40

Actual behavior: Cost needs to be re-calculated

Mileage version:Latest
Android version:2.2
Device: Desire

Please provide any additional information below.

```

Original issue reported on code.google.com by `DaveMcM...@gmail.com` on 17 Mar 2011 at 3:27

**Comment by GoogleCodeExporter** [comment]
```
Reproduced; thanks for the bug report. Will be fixed in the next update.
```

Original comment by `evancharlton` on 17 Apr 2011 at 7:33
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
http://code.google.com/p/android-mileage/source/detail?r=578d2d053fa7d730b590b7b
7274527c12bc9a9cc

Will be released in 3.1.0
```

Original comment by `evancharlton` on 17 Apr 2011 at 8:18
- Changed state: **Fixed**
